Commands.cs

```diff
@@ -19,6 +21,122 @@ public Commands(WarpplateManager manager, Player[] players)
             Players = players;
         }
 
+        public void MainCommand(CommandArgs args)
+        {
+            string cmd = "help";
+            if (args.Parameters.Count > 0)
+            {
+                cmd = args.Parameters[0].ToLower();
+            }
+            switch (cmd)
+            {
+                case "set":
+                    SetWarpplate(args);
+                    break;
+
+                case "del":
+                case "delete":
+                case "remove":
+                    DeleteWarpplate(args);
+                    break;
+
+                case "setdest":
+                case "setdestination":
+                    SetWarpplateDestination(args);
+                    break;
+
+                case "deldest":
+                case "deletedestination":
+                    RemoveWarpplateDestination(args);
+                    break;
+
+                case "info":
+                    WarpplateInformation(args);
+                    break;
+
+                case "list":
+                    ListWarpplates(args);
+                    break;
+
+                case "allow":
+                case "toggle":
+                    WarpplateAllow(args);
+                    break;
+
+                case "reload":
+                    ReloadWarpplates(args);
+                    break;
+
+                case "delay":
+                    SetWarpplateDelay(args);
+                    break;
+
+                case "width":
+                    SetWarpplateWidth(args);
+                    break;
+
+                case "height":
+                    SetWarpplateHeight(args);
+                    break;
+
+                case "resize":
+                case "size":
+                    SetWarpplateSize(args);
+                    break;
+
+                case "label":
+                case "display":
+                    SetWarpplateLabel(args);
+                    break;
+
+                case "dim":
+                case "dimension":
+                    SetWarpplateDimension(args);
```

Comment:
The permission has been changed from `warpplate.setdimensional` -> `warpplate.set` which seems unintentional given the permission check in the help section.

Reply:
Good catch, missed it while testing the plugin at 5am :)
Resolved in 49bfcc5d3d5519cb2a7056a8b3afc44faa2ee300